### ISSUE TITLE
Newly introduced test 125_shared_boolean.t requires threads::shared 1.21

### ISSUE THREAD

**Issue by ribasushi** [rurban/Cpanel-JSON-XS#170]
As per subject - the test needs to check for minimum version of t::s, and skip_all if not available.

Failure without this:
```
Undefined subroutine &main::shared_clone called at t/125_shared_boolean.t line 14.
# Looks like your test exited with 255 before it could output anything.
[06:52:22] t/125_shared_boolean.t ..... 
Dubious, test returned 255 (wstat 65280, 0xff00)
Failed 8/8 subtests 
```

Thanks!

**Comment by karenetheridge** [comment]
This affects all perls at or before 5.10.0.

**Comment by karenetheridge** [comment]
Ah indeed, I had not realized (having not ever really used threads much) that threads::shared is dual-life rather than core only.

**Comment by rurban** [comment]
Fixed with 4.22/4.23
